.github/workflows/test-docs.yml

```diff
@@ -0,0 +1,44 @@
+name: "Test Jupyter Notebooks"
```

Comment:
Maybe the filename could be `test-example-notebooks`? Will check what test_docs does...

Reply:
Renamed to `test-examples`